sdk/monitor/monitor-query/src/metricsClient.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { TokenCredential } from "@azure/core-auth";
+import {
+  MetricsListResponse,
+  MonitorManagementClient as GeneratedMetricsClient
+} from "./generated/metrics/src";
+import {
+  MetricDefinitionsListResponse,
+  MonitorManagementClient as GeneratedMetricsDefinitionsClient
+} from "./generated/metricsdefinitions/src";
+import {
+  MetricNamespacesListResponse,
+  MonitorManagementClient as GeneratedMetricsNamespacesClient
+} from "./generated/metricsnamespaces/src";
+import {
+  GetMetricDefinitionsOptions,
+  GetMetricNamespaces,
+  QueryMetricsOptions
+} from "./models/metricsModels";
+
+export class MetricsClient {
+  private _metricsClient: GeneratedMetricsClient;
+  private _definitionsClient: GeneratedMetricsDefinitionsClient;
+  private _namespacesClient: GeneratedMetricsNamespacesClient;
+
+  constructor(tokenCredential: TokenCredential) {
+    this._metricsClient = new GeneratedMetricsClient(tokenCredential, "2018-01-01", {});
```

Comment:
can we pull the api versions out into constants?

Reply:
Done. Using the very interestingly named auto-generated ones.